sdk/python/kfp/_client.py

```diff
@@ -618,8 +618,8 @@ def _create_job_config(self, experiment_id, params, pipeline_package_path, pipel
       params: A dictionary with key (string) as param name and value (string) as param value.
       pipeline_id: The id of a pipeline.
       version_id: The id of a pipeline version. 
-        If both pipeline_id and version_id are specified, pipeline_id will take precendence
-        This will change in a future version, so it is recommended to use version_id by itself.
+        If both pipeline_id and version_id are specified, version_id will take precendence.
+        If only pipeline_id is specified, the default version of this pipeline is used to create the run.
```

Comment:
Looks like there's a repeated comment that needs to be fixed as well:

https://github.com/kubeflow/pipelines/blob/759180537788e01a7906432255ad663c296e9518/sdk/python/kfp/_client.py#L573-L576

Reply:
done